SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -927,6 +927,38 @@ public PersonalRelationshipType GetPersonalRelationshipTypeByDescription(string
                 .FirstOrDefault(prt => prt.Description.ToLower() == description.ToLower());
         }
 
+        public Infrastructure.PersonalRelationship GetPersonalRelationshipById(long relationshipId)
+        {
+            return _databaseContext.PersonalRelationships
+                .FirstOrDefault(prt => prt.Id == relationshipId);
+        }
+
+        public void DeleteRelationships(Infrastructure.PersonalRelationship relationship)
```

Comment:
Looks like where missing tests for this method?

Reply:
Test added :)